.docs/content/2.guide/1.how-to/how-to-use-htc-mock.md

```diff
@@ -0,0 +1,72 @@
+# How to use HTC Mock?
+
+HTC Mock is a test tool for ArmoniK. It is used to generate tasks that will be runned by ArmoniK. It provides a lot of options to personalize the tasks.
+
+HTC Mock is used in CI to test ArmoniK automatically but it can also be used in development to generate realistic tasks.
+
+::alert{type="info"}
+To populate the database, you can also use some scripts to populate database. See [Populate database?](../populate-database.md)
```

Comment:
To populate the database, you can also use scripts.  See [Populate database](../populate-database.md)